src/pages/home/report/ReportActionsView.js

```diff
@@ -113,6 +118,7 @@ class ReportActionsView extends React.Component {
         // We have switched to a new report
         if (prevProps.reportID !== this.props.reportID) {
             this.reset(prevProps.reportID);
+            this.shouldShowUnreadActionIndicator = true;
```

Comment:
Was this a bug? I may have introduced this or broken this feature 🙈

Reply:
Merging master did break stuff, but adding this line fixed it 😆